src/studio/src/designer/backend/Languages/ini/en.ini

```diff
@@ -405,9 +408,11 @@ model = Model
 multiple_answers = Multiple answers
 name = Name
 no_item_selected = No item selected
+nullable = Can be null
```

Comment:
Are we expecting the model-designer to have knowledge of  `null`?
How about `Can be unanswered`

Reply:
Good question, do you have any thoughts on this @Febakke ? Instead of a checkbox saying "can be null" on the group

Reply:
Great point @mjulstein
I think your suggestion is better.
This might not be relevant, but can this be confusing in relation with required?

Reply:
@Febakke maybe. Suggest we use the "can be unanswered" for now and adjust if we get feedback on this.